src/views/Settings.vue

```diff
@@ -66,8 +66,11 @@ export default defineComponent({
           }
         })
       }
+      this.store.dispatch('picklist/findPickList')
```

Comment:
I think this should be done on completion of set facility with a note why it is done